runners/genlayer-py-std/src/genlayer/gl/_internal/eth.py

```
@@ -1,17 +1,15 @@
-__all__ = ('eth_contract',)
+__all__ = ('contract_interface',)
```

Ruff (0.11.9)

Undefined name `contract_interface` in `__all__`

(F822)